app/controllers/flowchart_controller.rb

```diff
@@ -1,4 +1,103 @@
 # frozen_string_literal: true
 
 class FlowchartController < ApplicationController
+  def create
+    @flowchart = Flowchart.new(JSON.parse(request.body.read))
+
+    unless @flowchart.valid?
+      render status: 400, json: { error: 'Invalid Flowchart params' }
+      return
+    end
+
+    @flowchart = Flowchart.create(JSON.parse(request.body.read))
+    root_node = FlowchartNode.create(flowchart_id: @flowchart.id, text: 'New Node', header: 'Options', is_root: true, deleted: false)
```

Comment:
I would expect that is is more user defined in terms of what they want in the root node. Like a user may choose to have it be similar to the `title` and `description` or they may not . For some flowchart tools Ive seen in the past, sometimes when you click to create a new node, it renders a new node with some generic text that you would edit anyways. This is the thought process I had here. I thought it might look weird to have a blank root node initially.